service/RTC/StandardVideoSettings.ts

```diff
@@ -74,6 +75,19 @@ export const STANDARD_CODEC_SETTINGS = {
     }
 };
 
+export const VIDEO_CODECS_BY_COMPLEXITY = {
+    'camera' : [
+        CodecMimeType.AV1,
+        CodecMimeType.VP9,
+        CodecMimeType.VP8
+    ],
+    'desktop' : [
```

Comment:
This order reads weird. Doesn't it represent "the codec I want to be in, in ideal conditions" ? In that case, I thought we'd want to be in AV1 specially for SS, no?

Reply:
The complexity order is from highest to lowest. If the current codec is AV1 for camera, we will switch to VP9 and then to VP8. Same with SS codecs. If we are already encoding in AV1 for SS, we move onto modifying lastN/receive resolution.